spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Graph.scala

```diff
@@ -90,7 +90,7 @@ class Graph[T: ClassTag](val inputs : Seq[ModuleNode[T]],
           .map(n => {
             n._2.fromIndex match {
               case Some(i) =>
-                if (i == 1 && n._1.element.output.isTensor) {
+                if (n._1.element.output == null || (i == 1 && n._1.element.output.isTensor)) {
```

Comment:
when will it be null

Reply:
when the current node is a merge, then one of its input is null